scripts/image-version.sh

```diff
@@ -4,9 +4,13 @@ source scripts/image_vars.src
 
 pinImageVersions(){
   echo "# $(date)"
-  for PREFIX in ${VAR_PREFIXES_ARR[@]}; do
-    local IMG_VER=$(getVarValue "${PREFIX}" _VER)
-    local IMG_VAR="${PREFIX}_VER"
+  for PREFIX in "${VAR_PREFIXES_ARR[@]}"; do
```

Comment:
Undo this change -- `${VAR_PREFIXES_ARR[@]}` should be treated as separate arguments.